src/nni_manager/training_service/kubernetes/frameworkcontroller/frameworkcontrollerTrainingService.ts

```diff
@@ -211,16 +211,28 @@ class FrameworkControllerTrainingService extends KubernetesTrainingService imple
             if (this.azureStorageClient === undefined) {
                 throw new Error('azureStorageClient is not initialized');
             }
+            let retryCount: number = 1;
```

Comment:
retryCount always equals to 1?

Reply:
Add config file for this kind of configuration.